zipkin-lens/src/models/Span.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import Annotation from './Annotation';
+import Endpoint from './Endpoint';
+
+// Same type as Span in the OpenApi/Swagger model https://zipkin.io/zipkin-api/#
```

Comment:
I wonder if anyone would appreciate us publishing these types at some point.

Though it also brought the idea of perhaps using zipkin-js for its model types

https://www.github.com/openzipkin/zipkin-js/tree/master/packages%2Fzipkin%2Findex.d.ts

Just random idea :)

Reply:
probably this one is more the point https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin/src/model.js

if anything we could match the types and put a link to it for now and make a note? That way the idea lingers more directly, and we can move that comment about the type into zipkin-js?

fwiw if doing a dep on the openapi type, I'd be more inclined to publish from the api project not zipkin-js.. we have a similar thing published for proto (though not generated..)